src/__tests__/pages/__snapshots__/Home.test.jsx.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
- [ ] [optional] @Paulinelali It would be better if you updated this snapshot it failing when running your tests but passes when updated.